src/test/data/descriptors/SegmentAnnotateMerge.xml

```diff
@@ -77,12 +77,12 @@
                     </array>
                 </value>
             </nameValuePair>
-            <nameValuePair>
```

Comment:
This override would fail the test case for the newer override spec. Do you want me to keep it for now and deal with the test case later, or just remove that portion?